### PR TITLE
test: local activity tests

### PR DESCRIPTION
**Pull request by cartcrom**
## Description
Adds tests for parseLocal.ts in the Mini Portfolio Activity tab.

### CODE REVIEW

File: src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ function parseMigrateCreateV3(
 ): Partial<Activity> {
   const baseCurrency = getCurrency(lp.baseCurrencyId, chainId, tokens)
   const baseSymbol = baseCurrency?.symbol ?? t`Unknown`
-  const quoteCurrency = getCurrency(lp.baseCurrencyId, chainId, tokens)
+  const quoteCurrency = getCurrency(lp.quoteCurrencyId, chainId, tokens)
```
Comment (cartcrom) [inline src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts:125]: found this bug while writing the tests

**Review by just-toby — APPROVED** [review]

File: src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts (hunk 1 above)
Comment (yyip-dev) [reply src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.ts:125]: yay

**Review by cmcewen — APPROVED** [review]
love more tests. i would also appreciate if you added one for failing to parse and either throwing or failing gracefully (not sure what intended behavior is)

File: src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.test.ts [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
+import { TransactionStatus as MockTxStatus } from 'graphql/data/__generated__/types-and-hooks'
 import { TokenAddressMap } from 'state/lists/hooks'
-import { WrappedTokenInfo } from 'state/lists/wrappedTokenInfo'
 import {
   ExactInputSwapTransactionInfo,
   ExactOutputSwapTransactionInfo,
   TransactionDetails,
-  TransactionType,
+  TransactionInfo,
+  TransactionType as MockTxType,
```
[the remark this answers is not in the document]
Comment (cartcrom) [reply src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.test.ts:11]: jest requires types referenced within mocks to be prefixed with "Mock", hence the renaming

File: src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.test.ts [hunk 3]
```diff
@@ -50,58 +47,195 @@ function mockSwapInfo(
...
+      mockChainId,
+    ],
+    [
+      { info, ...mockCommonFields(id, mockAccount2, MockTxStatus.Failed) } as unknown as TransactionDetails,
+      mockChainId,
+    ],
+  ]
+}
+
+const mockTokenAddressMap: TokenAddressMap = {
```
[the remark this answers is not in the document]
Comment (cartcrom) [reply src/components/AccountDrawer/MiniPortfolio/Activity/parseLocal.test.ts:92]: For this use-case there isn't too much of a difference but I think other tests could benefit from a utility like that

**Review by zzmp — APPROVED** [review]

**Pull request merged**